data/json/weather_type.json

```diff
@@ -0,0 +1,292 @@
+    "sound_category": 3,
+    "sun_intensity": 0,
+    "requirements": { "pressure_max": 996, "required_weathers": [ "rain" ] }
+  },
+  {
+    "id": "lightning",
+    "type": "weather_type",
+    "name": "Lightning Storm",
+    "color": "c_yellow",
+    "map_color": "h_yellow",
+    "glyph": "%",
+    "ranged_penalty": 4,
+    "sight_penalty": 1.25,
+    "light_modifier": -45,
+    "sound_attn": 8,
+    "dangerous": false,
+    "precip": 3,
+    "rains": true,
+    "acidic": false,
+    "effects": [ { "name": "thunder", "intensity": 50 }, { "name": "lightning", "intensity": 600 } ],
+    "tiles_animation": "weather_rain_drop",
+    "weather_animation": { "factor": 0.04, "color": "c_light_blue", "glyph": "," },
+    "sound_category": 3,
+    "sun_intensity": 0,
+    "requirements": { "pressure_max": 990, "required_weathers": [ "thunder" ] }
+  },
+  {
+    "id": "acid_drizzle",
+    "type": "weather_type",
+    "name": "Acidic Drizzle",
+    "color": "c_light_green",
+    "map_color": "c_yellow_green",
+    "glyph": ".",
+    "ranged_penalty": 2,
+    "sight_penalty": 1.03,
+    "light_modifier": -20,
+    "sound_attn": 1,
+    "dangerous": true,
+    "precip": 2,
+    "rains": true,
+    "acidic": true,
+    "effects": [ { "name": "light_acid", "intensity": 180 } ],
+    "tiles_animation": "weather_acid_drop",
+    "weather_animation": { "factor": 0.01, "color": "c_light_green", "glyph": "." },
+    "sound_category": 1,
+    "sun_intensity": 2,
+    "requirements": { "acidic": true, "required_weathers": [ "drizzle" ] }
+  },
+  {
+    "id": "acid_rain",
+    "type": "weather_type",
+    "name": "Acid Rain",
+    "color": "c_green",
+    "map_color": "c_yellow_green",
+    "glyph": "o",
+    "ranged_penalty": 4,
+    "sight_penalty": 1.1,
+    "light_modifier": -30,
+    "sound_attn": 4,
+    "dangerous": true,
+    "precip": 3,
+    "rains": true,
+    "acidic": true,
+    "effects": [ { "name": "acid", "intensity": 2 } ],
+    "tiles_animation": "weather_acid_drop",
+    "weather_animation": { "factor": 0.02, "color": "c_light_green", "glyph": "," },
+    "sound_category": 3,
+    "sun_intensity": 0,
+    "requirements": { "acidic": true, "required_weathers": [ "rain" ] }
+  },
+  {
+    "id": "flurries",
+    "type": "weather_type",
+    "name": "Flurries",
+    "color": "c_white",
+    "map_color": "c_dark_gray_cyan",
+    "glyph": ".",
+    "ranged_penalty": 2,
+    "sight_penalty": 1.12,
+    "light_modifier": -15,
+    "sound_attn": 2,
+    "dangerous": false,
+    "precip": 2,
+    "rains": false,
+    "acidic": false,
+    "tiles_animation": "weather_snowflake",
+    "weather_animation": { "factor": 0.01, "color": "c_white", "glyph": "." },
+    "sound_category": 4,
+    "sun_intensity": 1,
+    "requirements": { "temperature_max": 33, "required_weathers": [ "drizzle" ] }
+  },
+  {
+    "id": "snowing",
+    "type": "weather_type",
+    "name": "Snowing",
```

Comment:
You need to add code to "lang/extract_json_strings.py" in order to extract this string for translation. I assume it's enough to add the "type" value to the `automatically_convertible` list.